src/punycode.c

```diff
@@ -402,7 +403,7 @@ int main (int argc, char **argv)
       if (r == EOF || r == 0)
          break;
 
-      if (r != 2 || uplus[1] != '+' || codept > (DWORD) - 1)
```

Comment:
Why? What do you think could happen on Cygwin-x64 here?

Reply:
Oh sorry 64-bit Windows exception, need to be fixed.